lib/main.coffee

```diff
@@ -2,6 +2,17 @@
 helpers = require 'atom-linter'
 path = require 'path'
 
+# This function returns true if version v2 is greater than v1
```

Comment:
actually with the new function IMO the comment isn't needed anymore, this just tells the same as the name does; before I asked for it because the function name wasn't so obvious. That's a good sign for the design of the functionality :+1: